src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -75,6 +83,58 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     this.setState({ showCreatePipelineRunModal });
   };
 
+  handleChange = event => {
+    const inputValue = event.target.value;
+    this.setState({
+      currentFilterValue: inputValue
+    });
+  };
+
+  handleAddFilter = () => {
+    this.setState({
```

Comment:
you might have some unexpected behaviour around the number of rer-enders happening here

Reply:
hmm, It seemed to work fine in practice and in the tests, what behaviour do you think I need to address?

Reply:
If I don't reset the state for the notification it stays their indefinitely even after the user inputs a new valid filter, I'm not sure of a way that I can improve this?

Reply:
Not needed sure, I was thinking https://github.com/tektoncd/dashboard/pull/427/files#diff-ee9adb63dcebb1625cd7ef8aa5d27cb4R111 or in resetCurrentFilter it could unset that value

Reply:
ah got you, I'll go with the latter